src/scss/theme/index.scss

```diff
@@ -16,6 +16,6 @@
 @import "../utilities/index";
 @import 'background-classes';
 @import 'button';
-
+@import "_cta-list";
```

Comment:
shouldn't this just be `@import "cta-list"` ?

Reply:
yes it should.

Reply:
@imonroe fixed it.